src/rust/dbv2/migrations/2023-12-21-132732_annotations/up.sql

```diff
@@ -0,0 +1,32 @@
+-- Your SQL goes here
+CREATE TABLE annotations (
+    "time" timestamptz NOT NULL,
+    "timeEnd" timestamptz,
+    "title" text NOT NULL,
+    "text" text,
+    "tag" text NOT NULL,
+);
+
+INSERT INTO annotations ("time", "title", "tag") VALUES (
+    '2023-11-28 14:03:00+00'::timestamptz,
+    'Econia mainnet launch',
+    'Econia'
+);
```

Comment:
I don't think migrations is the right place for the actual dataset

What about a separate json file?

Then we can do a CI/CD script that just loads the data from a git commit, etc.

Maybe the JSON file could be at `/src/rust/dbv2/grafana-annotations.json?`